docs/src/maintain/manage-releases.md

```diff
@@ -13,55 +13,61 @@ Releases are when a project formally publishes a new version so the community ca
 1. Add the `patch release pending` label to the release issue. (When this label is present, `eslint-github-bot` will create a pending status check on non-semver-patch pull requests, to ensure that they aren't accidentally merged while a patch release is pending.)
 
-On the Monday following the scheduled release, the release team needs to determine if a patch release is necessary. A patch release is considered necessary if any of the following occurred since the scheduled release:
+All release-related communications occur in the `#team` channel on Discord.
+
+On the Monday following the scheduled release, the release manager needs to determine if a patch release is necessary. A patch release is considered necessary if any of the following occurred since the scheduled release:
 
 * A regression bug is causing people's lint builds to fail when it previously passed.
 * Any bug that is causing a lot of problems for users (frequently happens due to new functionality).
 
 The patch release decision should be made as early on Monday as possible. If a patch release is necessary, then follow the same steps as the scheduled release process.
 
-In rare cases, a second patch release might be necessary if the release is known to have a severe regression that hasn't been fixed by Monday. If this occurs, the release team should announce the situation on the release issue, and leave the issue open until all patch releases are complete. However, it's usually better to fix bugs for the next release cycle rather than doing a second patch release.
+In rare cases, a second patch release might be necessary if the release is known to have a severe regression that hasn't been fixed by Monday. If this occurs, the release manager should announce the situation on the release issue, and leave the issue open until all patch releases are complete. However, it's usually better to fix bugs for the next release cycle rather than doing a second patch release.
 
 After the patch release has been published (or no patch release is necessary), close the release issue and inform the team that they can start merging in semver-minor changes again.
 
 ## Emergency Releases
 
-In general, we try not to do emergency releases (an emergency release is unplanned and isn't the regularly scheduled release or the anticipated patch release). Even if there is a regression, it's best to wait the weekend to see if any other problems arise so a patch release can fix as many issues as possible.
+An emergency release is unplanned and isn't the regularly scheduled release or the anticipated patch release.
+
+In general, we try not to do emergency releases. Even if there is a regression, it's best to wait the weekend to see if any other problems arise so a patch release can fix as many issues as possible.
```

Comment:
```suggestion
In general, we try not to do emergency releases. Even if there is a regression, it's best to wait until the weekend to see if any other problems arise so a patch release can fix as many issues as possible.
```

Reply:
Makes sense, I'll let @bpmutter take a call on this one.

Reply:
```suggestion
In general, we try not to do emergency releases. Even if there is a regression, it's best to wait until Monday to see if any other problems arise so a patch release can fix as many issues as possible.
```